final_task/README.md

```diff
@@ -1,3 +1,69 @@
-# Your readme here
-Some text.
-Checkout how to write this file using *markdown*.
+
+This program which receives RSS URL and prints results in human-readable
+format.
+
+positional arguments:
+  source         RSS URL
+
+optional arguments:
+  -h, --help     show this help message and exit
+  --version      Print version info
+  --json         Print result as JSON in stdout
+  --verbose      Outputs verbose status messages
+  --limit LIMIT  Limit news topics if this parameter provided
+
+
+
+Installation recommendation rss-reader:
+1)Open terminal 
+2)Enter "pip install setuptools" or "pip3 install setuptools"
+3)Go to the folder final_task
+4)Enter "python3 setup.py install"
+5)Application installed
+6)To run the utility, type in the terminal "rss-reader" then a space and url on news
+Example : rss-reader  https://news.yahoo.com/rss
+
+
+To use caching you must have a postgresql database on your computer or laptop
+with default parameters:
+    database="postgres",
+    user="postgres",
+    password="1",
+    host="localhost",
+    port="5432"    
+If you do not, follow these commands:    
+1)Open terminal
+2)sudo apt-get install postgresql
+3)sudo -u postgres psql
+4)\password
+5)1
+6)1
+7)\conninfo find out the port and if it is not equal 5432, change the config.txt port to your
```

Comment:
As you can see from our struggle, making your standard user to perform some additional steps like setting up the database / changing some parameters by his own hands is a bad idea for a project distribution.

In the real production all these steps are usually performed by bash scripts or inside docker containers so the user only need to write 1-2 lines in the terminal to get the things going.

It is ok to use postgres db if you are aiming for iteration 6, but ideally the process should be fully automated, so the user do not need to install/change any additional stuff